Remove unused expectations of SVGCursorElement-svgdom-* tests

Ahmad-S792 opened the pull request:
#### b85b96c50d057a446880f433c5fe9404e7152094
<pre>
Remove unused expectations of SVGCursorElement-svgdom-* tests

<a href="https://bugs.webkit.org/show_bug.cgi?id=269339">https://bugs.webkit.org/show_bug.cgi?id=269339</a>

Reviewed by Nikolas Zimmermann.

This patch removes unused file expectations for following tests (which don&apos;t exist anymore):

&gt; svg/dynamic-updates/SVGCursorElement-svgdom-x-prop.html:
&gt; svg/dynamic-updates/SVGCursorElement-svgdom-y-prop.html:

* LayoutTests/platform/gtk/svg/dynamic-updates/SVGCursorElement-svgdom-x-prop-expected.png: Deleted
* LayoutTests/platform/gtk/svg/dynamic-updates/SVGCursorElement-svgdom-y-prop-expected.png: Ditto
* LayoutTests/platform/mac/svg/dynamic-updates/SVGCursorElement-svgdom-x-prop-expected.png: Ditto
* LayoutTests/platform/mac/svg/dynamic-updates/SVGCursorElement-svgdom-y-prop-expected.png: Ditto

Canonical link: <a href="https://commits.webkit.org/274622@main">https://commits.webkit.org/274622@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/f9449a98b2a86a3b57f2e143dc16af6638787ff0

| Misc | iOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/39573 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/48/builds/18552 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/14/builds/41929 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/42108 "Built successfully") | [✅ 🛠 wincairo](https://ews-build.webkit.org/#/builders/32/builds/35473 "Built successfully") 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/49/builds/21441 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/51/builds/15881 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/33051 "Passed tests") | 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/40147 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/47/builds/15636 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/34252 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/13572 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/42/builds/13548 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/10/builds/35205 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/43385 "Built successfully") | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/35926 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/36/builds/35536 "Passed tests") | [❌ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/39330 "Found 1 new test failure: http/tests/inspector/network/har/har-page.html (failure)") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/44/builds/14385 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/50/builds/11848 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/37599 "Passed tests") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/46/builds/15991 "Built successfully") | | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/8869 "Built successfully and passed tests") | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/43/builds/16039 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/45/builds/15648 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

nikolaszimmermann reviewed (approved):
r=me, thanks @Ahmad-S792

Ahmad-S792 committed:
Remove unused expectations of SVGCursorElement-svgdom-* tests

https://bugs.webkit.org/show_bug.cgi?id=269339

Reviewed by Nikolas Zimmermann.

This patch removes unused file expectations for following tests (which don't exist anymore):

> svg/dynamic-updates/SVGCursorElement-svgdom-x-prop.html:
> svg/dynamic-updates/SVGCursorElement-svgdom-y-prop.html:

* LayoutTests/platform/gtk/svg/dynamic-updates/SVGCursorElement-svgdom-x-prop-expected.png: Deleted
* LayoutTests/platform/gtk/svg/dynamic-updates/SVGCursorElement-svgdom-y-prop-expected.png: Ditto
* LayoutTests/platform/mac/svg/dynamic-updates/SVGCursorElement-svgdom-x-prop-expected.png: Ditto
* LayoutTests/platform/mac/svg/dynamic-updates/SVGCursorElement-svgdom-y-prop-expected.png: Ditto

Canonical link: https://commits.webkit.org/274622@main
